src/main/java/core/basesyntax/service/FileWriterService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.nio.file.Path;
+
+public interface FileWriterService {
+    void writeToFile(Path path, String report);
```

Comment:
let's pass not object path here, but its String representation and inside of this method create an instance of Path based on this String representation